lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -1,39 +1,28 @@
 """
-axis_artist.py module provides axis-related artists. They are
+=========================
+Artists for axis elements
```

Comment:
I'll have to check again, but my intuition says we shoud not have headings in docstrings.

Reply:
indeed